DP02_17_Galaxy_Photometry.ipynb

```diff
@@ -0,0 +1,1077 @@
+{
```

Comment:
<p>Maybe in the caption mention that histogram has the same color as the corresponding scatter, as there's no label for the histograms.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<ol><li>Thanks! I clarified that. 2. I think one can't effectively measure a size for something smaller than 2 pixels.</li></ol>